apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/index.js

```diff
@@ -26,6 +31,40 @@ import mapBlocksRecursively from './utils/map-blocks-recursively';
 
 const DEFAULT_HOMEPAGE_TEMPLATE = 'maywood';
 
+/*
+ * Register a `a8c/spt-template-post-title` used only
+ * for the Large Preview of the template.
+ * This block will be unregistered once the template is inserted,
+ * the component unmounted, modal closed, etc...
+ */
+registerBlockType( 'a8c/spt-template-post-title', {
```

Comment:
I think maybe we should add some context as to _why_ we're adding a custom block here. You provided reasoning in the PR comments which would work well.